README.md

```diff
@@ -22,7 +22,7 @@ For contributing, checkout the [contribution guidelines][contribution docs] and
 
 [setup docs]: https://docs.datadoghq.com/tracing/setup/ruby/
 [api docs]: https://github.com/DataDog/dd-trace-rb/blob/master/docs/GettingStarted.md
-[gem docs]: https://s3.amazonaws.com/gems.datadoghq.com/trace/docs/index.html
+[gem docs]: https://datadog.github.io/dd-trace-rb/
```

Comment:
Until we push to the release branch, this URL will 404. Should we change the README on a separate PR?

Reply:
Never mind. I see we can also trigger the workflow manually 😄 

We just need to remember to run after merging